test/fixtures/vars.yml

```diff
@@ -0,0 +1,106 @@
+        admin_ccs_category_email: __smoke_test_variables____production____admin_ccs_category_email__
+        admin_ccs_category_password: __smoke_test_variables____production____admin_ccs_category_password__
+        admin_ccs_sourcing_email: __smoke_test_variables____production____admin_ccs_sourcing_email__
+        admin_ccs_sourcing_password: __smoke_test_variables____production____admin_ccs_sourcing_password__
+        admin_manager_email: __smoke_test_variables____production____admin_manager_email__
+        admin_manager_password: __smoke_test_variables____production____admin_manager_password__
+        admin_framework_manager_email: __smoke_test_variables____production____admin_framework_manager_email__
+        admin_framework_manager_password: __smoke_test_variables____production____admin_framework_manager_password__
+functional_test_variables:
+    preview:
+        buyer_email: __functional_test_variables____preview____buyer_email__
+        buyer_password: __functional_test_variables____preview____buyer_password__
+        supplier_email: __functional_test_variables____preview____supplier_email__
+        supplier_password: __functional_test_variables____preview____supplier_password__
+        supplier2_email: __functional_test_variables____preview_____email__
+        supplier3_email: __functional_test_variables____preview_____email__
+        admin_email: __functional_test_variables____preview____admin_email__
+        admin_ccs_sourcing_email: __functional_test_variables____preview____admin_ccs_sourcing_email__
+        admin_ccs_category_email: __functional_test_variables____preview____admin_ccs_category_email__
+        admin_manager_email: __functional_test_variables____preview____admin_manager_email__
+        admin_framework_manager_email: __functional_test_variables____preview____admin_framework_manager_email__
+        admin_password: __functional_test_variables____preview____admin_password__
+jenkins_slack_url: __jenkins_slack_url__
+jenkins_admin_user: __jenkins_admin_user__
+jenkins_admin_password: __jenkins_admin_password__
+jenkins_api_user: __jenkins_api_user__
+jenkins_api_user_github_personal_access_token: __jenkins_api_user_github_personal_access_token__
+jenkins_build_monitor_user: __jenkins_build_monitor_user__
+jenkins_github_auth_by_hostname:
+    ci.marketplace.team:
+        client_id: __jenkins_github_auth_by_hostname____team____client_id__
+        client_secret: __jenkins_github_auth_by_hostname____team____client_secret__
+jenkins_github_email: __jenkins_github_email__
+java_certificate_store_pass: __java_certificate_store_pass__
+google_username: __google_username__
+google_password: __google_password__
+jenkins_gdrive_refresh_token: __jenkins_gdrive_refresh_token__
+jenkins_gdrive_db_dumps_folder_id: __jenkins_gdrive_db_dumps_folder_id__
+jenkins_gdrive_csv_data_export_folder_id: __jenkins_gdrive_csv_data_export_folder_id__
```

Comment:
What were the "tweaks by hand" & were they significant? I ask because it would be nice if it could generate this on the fly so we don't problems with divergence...

Reply:
They were fairly significant. I'm working on it, but I didn't want to delay this PR any longer.

Reply:
Python script is now included